framework/src/main/java/org/checkerframework/framework/type/ElementAnnotationApplier.java

```diff
@@ -132,7 +155,11 @@ private static void applyInternal(
      */
     public static void annotateSupers(
             List<AnnotatedDeclaredType> supertypes, TypeElement subtypeElement) {
-        SuperTypeApplier.annotateSupers(supertypes, subtypeElement);
+        try {
+            SuperTypeApplier.annotateSupers(supertypes, subtypeElement);
+        } catch (UnexpectedAnnotationLocationException e) {
+            throw new BugInCF(e.getMessage());
```

Comment:
Why is this a bug in the CF? Isn't it the same kind of user error as on line 92 above?